### PR TITLE
Update master version to 2.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by reswqa**

### CODE REVIEW

**Review by 1996fanrui — APPROVED** [review]
Thanks @reswqa for the updating!

LGTM assuming the CI is green.

File: .github/workflows/nightly-trigger.yml
Comment (XComp) [inline .github/workflows/nightly-trigger.yml:35]: Don't remove the 1.18 release branch just, yet. There might be another 1.18.2 being released before 1.18 finally reaching end-of-life.

**Review by XComp — CHANGES_REQUESTED** [review]
I know that we agreed on 1.20 being the last minor 1.x release. But did you reconfirm with the 2.0 release managers on the state of the 2.0 work? That might be worth doing before merging this PR to `master`. ...just to be sure.

**Pull request merged**
